### DISCUSSION TITLE
m3u: commentstring is empty

### DISCUSSION THREAD

**Discussion by avidseeker** [neovim/neovim#29580] [Q&A]
```sh
nvim test.m3u
:set commentstring
commentstring=
```
Expected: `commentstring=# %s`

Is this a bug in Neovim? Or is Neovim delegating this task to another plugin? In that case which plugin? treesitter? Where should I report this to?

**Comment by zeertzjq** [comment]
It's not a bug. It should be reported to Vim as a feature request.

**Comment by avidseeker** [comment]
https://github.com/vim/vim/issues/15156
